doc/user/content/cli/_index.md

```diff
@@ -19,8 +19,8 @@ Flag | Default | Modifies
 [`--processes`](#horizontally-scaled-clusters) | 1 | Number of coordinating Materialize nodes
 [`--tls-cert`](#tls-encryption) | N/A | Path to TLS certificate file
 [`--tls-key`](#tls-encryption) | N/A | Path to TLS private key file
-[`--workers`](#worker-threads) | 1 | Dataflow worker threads
-[`--w`](#worker-threads) | 1 |  Dataflow worker threads
+[`--workers`](#worker-threads) | REQ | Dataflow worker threads
+[`--w`](#worker-threads) | REQ |  Dataflow worker threads
```

Comment:
I think this still says `--w`?

Reply:
🤦 

Not sure how you feel about the "REQ" word there, I couldn't think of a better way to say that there is no default, but it seems kinda ugly.

Reply:
I'm of the mind that a lot of things can make sense to people; I say just go with this and we'll hear from people if they're confused.